### PR TITLE
Game model

### PR DESCRIPTION
**Pull request by reneesandefur**
Game model, 1 test, working on further, updated database.yml because unicode wasn't working with postgres(indicated template0), factory girl gem

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ development:
 test:
   <<: *default
   database: WarAndPiece_test
-
+  template: template0
```
Comment (rosalinekarr) [inline config/database.yml]: You probably don't want to commit these changes. The `database.yml` file shouldn't really need to change.

File: app/models/game.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Game < ApplicationRecord
+  belongs_to :player
```
Comment (rosalinekarr) [inline app/models/game.rb]: A game will need two players, one white and one black. Maybe add something like this instead:

```
belongs_to :black_player
belongs_to :white_player
```

File: spec/models/game_spec.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Game, type: :model do
+  it "should populate the board with pieces when started" do
+    game = FactoryGirl.create(:game)
+    expect(game.pieces.count).to eq 32
+  end
```
Comment (rosalinekarr) [inline spec/models/game_spec.rb]: Let's wait for Gary to start the "Add a Piece Model" task before adding this test.

File: config/database.yml (hunk 1 above)
Comment (reneesandefur) [reply config/database.yml]: How do I make this change consistent on my local machine without pushing it? Almost everything I did would gave me errors before I made this change

File: config/database.yml [hunk 4]
```diff
@@ -58,7 +58,7 @@ development:
 test:
   <<: *default
   database: WarAndPiece_test
-
+  #template: template0
```
Comment (rosalinekarr) [inline config/database.yml]: Let's remove these lines from this file.

Comment (reneesandefur) [reply config/database.yml]: I pulled it out but do I just keep adding it? i just get error without it

**Pull request merged**